xrpl/constants.py

```diff
@@ -1,6 +1,6 @@
 """Collection of public constants for XRPL."""
+import re
 from enum import Enum
-from re import compile
```

Comment:
Why not import the specific function? (Seems like that's what we normally do)

Reply:
Nevermind, I see there's a different standard in the other files you edited.

Reply:
yeah, did that because imo in this case, like with the `json` package, it's not at all clear to see:

```python
from re import compile

.....

regex = compile("[A-Z]")
```

i think you want to see the full `re.compile`